src/main/scala/com/github/tarao/namedcap/MultiMap.scala

```diff
@@ -19,14 +19,17 @@ class MultiMap(m: Map[String, Seq[String]])
     extends ImmutableMap[String, String] {
   def get(key: String): Option[String] = m.get(key).flatMap(_.headOption)
   def getAll(key: String): Seq[String] = m.getOrElse(key, Seq.empty)
-  def +[V >: String](kv: (String, V)): MultiMap = {
-    val (key, value) = kv
-    val list = m.getOrElse(key, Vector.empty) :+ value.asInstanceOf[String]
-    new MultiMap(m + (key -> list))
-  }
-  def -(key: String): MultiMap = new MultiMap(m - key)
```

Comment:
`-` is marked as `final` in Scala 2.13. We cannot override it.

c.f. https://www.scala-lang.org/api/2.13.8/scala/collection/immutable/Map.html